BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/security/service/SecurityUtil.java

```diff
@@ -82,4 +82,27 @@ String createToken(User user) {
                 .signWith(SignatureAlgorithm.HS512, tokenHash)
                 .compact();
     }
+
+    // TODO: Extract instance in safe place and disable using custom url.
+    String getInstanceUrl(String instanceKey, String path) {
+        switch (instanceKey) {
+            // DEV
+            case "975dd2ca-58eb-407b-ba0f-858f15f7304d":
```

Comment:
Wouldn't it be better to use properties for this? Or is precisely that what the TODO is about?

Reply:
yes that is the TODO, I will fix this once integration with UI works